cli/src/main.rs

```diff
@@ -126,12 +126,17 @@ fn main() {
 
 	debug!("command = {:?}", command);
 
-	if cfg!(target_os = "windows") {
-		Command::new("cmd").args(["/C", command.as_str()]).output().expect("failed to execute process");
+	let status = if cfg!(target_os = "windows") {
+		Command::new("cmd").args(["/C", command.as_str()]).output().expect("failed to execute process").status
 	} else {
-		let _ =
-			Command::new("sh").arg("-c").arg(command).spawn().expect("failed to execute process").wait_with_output();
-	}
+		match Command::new("sh").arg("-c").arg(command).spawn().expect("failed to execute process").wait_with_output() {
+			Ok(result) => result.status,
+			_ => std::process::exit(1),
```

Comment:
This is a bit hacky, but it is cleaner than the other ways I found and pulls in less stuff.